CHANGELOG.md

```diff
@@ -11,7 +11,8 @@ Note that Sockeye has checks in place to not translate with an old model that wa
 Each version section may have have subsections for: _Added_, _Changed_, _Removed_, _Deprecated_, and _Fixed_.
 
 ## [1.18.6]
-- Added support for LHUC
+- Added support for LHUC (David Vilar, "Learning Hidden Unit Contribution for
```

Comment:
This is missing `### Added` section. I think it would be great to have a little bit more detail on this here, or we could add a tutorial under `tutorials/` for this. This is a nice feature and people should know how to use it.